From 5cd3db1eeac44a77f508e0b3db15764ed26b2499 Mon Sep 17 00:00:00 2001
From: Thor Thayer <thor.thayer@linux.intel.com>
Date: Fri, 8 Nov 2019 16:40:53 -0600
Subject: [PATCH 013/120] arm64: dts: agilex: Add SysMgr compatible

commit 9341789ce83a899d0f956b369001d35d2103a756 from
https://github.com/altera-opensource/linux-socfpga.git

[backport 'commit 5dfd391e10a0 ("arm64: dts: agilex: Add SysMgr compatible")']
from dinguyen/for_next_v5.5_socfpga_dts_v2

Add the System Manager compatible string to the sysmgr node
for accessing the System Manager driver.
Ethernet and EDAC drivers use this driver.

Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 687ae8a66409..ef9a239af143 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -399,7 +399,7 @@
 		};
 
 		sysmgr: sysmgr@ffd12000 {
-			compatible = "altr,sys-mgr", "syscon";
+			compatible = "altr,sys-mgr-s10","altr,sys-mgr";
 			reg = <0xffd12000 0x500>;
 		};
 
-- 
2.31.1

